testutil/host.go

```diff
@@ -61,11 +61,18 @@ type EphemeralContractor struct {
 
 	accounts map[proto4.Account]types.Currency
 
-	mu sync.Mutex
+	mu       sync.Mutex
+	shutdown chan struct{}
 }
 
 var _ rhp4.Contractor = (*EphemeralContractor)(nil)
 
+// Close closes the contractor by interrupting its background loop
+func (ec *EphemeralContractor) Close() error {
```

Comment:
FYI minor change to avoid 200 goroutines in a stacktrace of a CI job running the full test suite